main.c

```diff
@@ -30,17 +30,19 @@ static struct slurp_output *output_from_surface(struct slurp_state *state,
 	struct wl_surface *surface);
 
 static void move_seat(struct slurp_seat *seat, wl_fixed_t surface_x,
-		wl_fixed_t surface_y) {
-	int x = wl_fixed_to_int(surface_x) + seat->current_output->logical_geometry.x;
-	int y = wl_fixed_to_int(surface_y) + seat->current_output->logical_geometry.y;
+	wl_fixed_t surface_y,
+	struct slurp_selection *current_selection) {
```

Comment:
Style: indent arguments with one extra tab to be able to tell them apart from the function body